### ISSUE TITLE
NMS.GEN.LANG.004 - language determination for user with unknown location

### ISSUE THREAD

**Issue by larubbio** [motech-implementations/mim#122]
If the location information of FLW/MCTS-beneficiary is not known corresponding to the received MSISDN number, then following rules for language determination apply:
- **Case A**: If the circle information cannot be determined, the user is prompted in the national default language by the IVR to enter their Language-location code via DTMF.
- **Case B**: If the circle information can be determined and 
 - **Case B1**: there is only one “Language-location code” possible in the circle (based on geographical coverage of the circle), then the corresponding “Language-location code” shall be used for the received MSISDN number. 
 - **Case B2**: there are multiple languages possible in the circle (based on geographical coverage of the circle), the user is prompted in the circle’s default language by the IVR to enter their Language-location code via DTMF. 

Case B2 covers two types of cases. One case is of north-east where there are multiple states in a circle and consequently multiple languages. Another case is that where multiple languages are used within a state. Both cases are covered in this scenario.

**Comment by llavoie** [comment]
Right now our API rejects calls which omit the Circle.

**Comment by larubbio** [comment]
This is dependent on #119

**Comment by koshalt** [comment]
Does this need to be open for the issue outlined in the PR?

**Comment by larubbio** [comment]
I opened a new ticket to track that.

**Comment by koshalt** [comment]
ok, just wanted to make sure in case you were planning to link them :)

**Comment by larubbio** [comment]
Good point.  Case B1 is under discussion with IMI about who should implement it.  That discussion is being tracked in #252
